Tests/UnitTests/ConnectLibraryTests/ConnectTests/ProtocolClientConfigTests.swift

```diff
@@ -213,4 +213,46 @@ final class ProtocolClientConfigTests: XCTestCase {
             unaryGET: .disabled
         ).shouldUseUnaryGET(for: request))
     }
+
+    func testTransformToGETIfNeededWithRequestCompression() {
```

Comment:
I think it'd make sense for this to be split into 2 tests (one with compression and one without)